build/controller_primary.sh

```diff
@@ -127,25 +156,25 @@ popd
 
 # here we run ansible using the run-playbooks script in the ansible repo
 if [ "%%RUN_ANSIBLE%%" = "True" ]; then
-  cd ${checkout_dir}/rpc-openstack/os-ansible-deployment
-  scripts/run-playbooks.sh
-  pushd ${checkout_dir}/rpc-openstack/rpcd/playbooks
-    openstack-ansible repo-build.yml
-    openstack-ansible repo-pip-setup.yml
-    if [ "$DEPLOY_MONITORING" = "yes" ] && [ "$TEST_MONITORING" = "yes" ]; then
-      openstack-ansible "test-maas.yml" --tags "setup,setup-fake-hp"
-    fi
-    if [ "$DEPLOY_MONITORING" = "yes" ]; then
-      openstack-ansible setup-maas.yml
-    fi
+  pushd ${checkout_dir}/rpc-openstack
+    export DEPLOY_HAPROXY="yes"
+    export DEPLOY_OSAD=$DELOY_OPENSTACK
+    export DEPLOY_ELK=$DEPLOY_LOGGING
+    export DEPLOY_MAAS=$DEPLOY_MONITORING
+    export DEPLOY_TEMPEST=$DEPLOY_TEMPEST
```

Comment:
I noticed that this line is not in config_controller_primary.sh.  Did config_controller_primary.sh not get rebuilt correctly?